app/src/main/res/values-en/strings.xml

```diff
@@ -719,12 +719,12 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <string name="quest_buildingType_bungalow_description2">small detached house (summer house, holiday cottage, cabin …)</string>
     <string name="quest_buildingType_static_caravan">Mobile home</string>
     <string name="quest_buildingType_commercial">Commercial building</string>
-    <string name="quest_buildingType_commercial_generic_description">building where people work, shop or do any other commercial activities</string>
+    <string name="quest_buildingType_commercial_generic_description">building where people work, shop or do other commercial activities</string>
     <string name="quest_buildingType_industrial">Industrial building</string>
     <string name="quest_buildingType_industrial_description">e.g. a factory, workshop, car repair garage, …</string>
     <string name="quest_buildingType_office">Office building</string>
     <string name="quest_buildingType_retail">Shop(s)</string>
-    <string name="quest_buildingType_retail_description">including buildings for restaurants, cafés, big stores …</string>
+    <string name="quest_buildingType_retail_description">including buildings for restaurants, cafés, big-box stores …</string>
```

Comment:
I guess one could just say "buildings for .... stores". After all, also small shops can have a dedicated building.

Reply:
Large? Might avoid confusion between big-box stores. (I'm not an en-us speaker)

Reply:
Is this intended to mean a building dedicated to a single store? We call those “standalone stores” in American English.

Reply:
Oh you meant standalone stores.  Yes, that was definitely not what I got from the original text.

Reply:
Why does it need to be "big" stores?  Can't there be a "small" building used for retail?

Reply:
Yes, it could also be a small building. However, if it is very small, it's likely a `shop=kiosk` then right? (e.g. a newspaper stand or something of similar size)

Reply:
@westnordost I am thinking of buildings of this size:
![image](https://user-images.githubusercontent.com/3254090/168496925-40ce89b4-aa2c-422f-a85d-8d690643184c.png)

Nobody would call this a kiosk, and nobody would call this a big store.

Reply:
Or are you saying that "this is a house repurposed for retail"?

Reply:
it looks retail to me, yeah, and not a kiosk

Reply:
Can we just remove "big" and say "stores" ?

Reply:
ok